lib/test/Assertions.js

```diff
@@ -83,6 +83,14 @@ const clear = repo => {
   };
 };
 
+const count = repo => {
```

Comment:
generic assertions are run for all repository implementations, so this should be moved to MongooseRepository.test.js if its only going to exist for mongoose